src/libraries/Common/tests/TestUtilities/TestEventListener.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+using Xunit.Abstractions;
+
+namespace TestUtilities;
+
+/// <summary>
+/// Logging helper for tests.
+/// Logs event source events into test output.
+/// </summary>
+public sealed class TestEventListener : EventListener
+{
+    private readonly ITestOutputHelper _output;
+    private readonly HashSet<string> _sourceNames;
+
+    // Until https://github.com/dotnet/runtime/issues/63979 is solved.
+    private List<EventSource> _eventSources = new List<EventSource>();
+
+    public TestEventListener(ITestOutputHelper output, params string[] sourceNames)
```

Comment:
Can we please also have an overload that uses Console instead of ITestOutputHelper?

Reply:
Added.